products/ansible.md

```diff
@@ -1,6 +1,6 @@
 ---
 permalink: /ansible
-title: Ansible
+title: Red Hat Ansible
```

Comment:
I think this has the same problem as opensearch. RH Ansible is [something else](https://www.redhat.com/en/technologies/management/ansible), which we don't track.

Reply:
Agree with @captn3m0, but not for the same reasons. The "something else" is "Red Hat Ansible Automation Platform", not just "Red Hat Ansible".

Having said that Red Hat never refers to Ansible using "Red Hat Ansible", it is always just "Ansible" (see https://www.ansible.com/integrations/infrastructure/red-hat for example).